packages/core/src/modules/member/use-cases/has-review-access.ts

```diff
@@ -0,0 +1,30 @@
+import { db } from '@oyster/db';
+
+export async function hasReviewAccess(id: string): Promise<boolean> {
```

Comment:
Let's change the parameter name from `id` to `studentId` just so it's more clear what id we are referring to